man/readtags.1.rst.in

```diff
@@ -306,6 +338,72 @@ A quick read tells us: If $-entry has "file" kind, and &-entry doesn't, the
 sorter becomes ``(<> 1 -1)``, which produces ``1``, so the $-entry is put below
 the &-entry, exactly what we want.
 
+Formatting
+~~~~~~~~~~
+A formatter expression defines how readtags prints tag entries.
+
+A formatter expression may produce a string, a boolean, an integer,
+or a list. Readtags prints the produced string, and integer as is.
+Readtags prints nothing for ``#f``, and a newline for ``#t``.
+
+Readtags expects a list contains any number of strings, booleans,
```

Comment:
Does the whole formatter expression needs to be a list? or it can be a string/boolean/integer? If the latter is true, I'd suggest:

```suggestion
A list could contain any number of strings, booleans,
```

If the former is true, then the description in the previous paragraph is not accurate.

Reply:
You are correct. The latter is true. Thank you.